packages/sirius-web/frontend/sirius-web-view-fork/src/contributions/ForkViewMenuAction.tsx

```diff
@@ -92,12 +92,10 @@ export const ForkViewMenuAction = ({ editingContextId, representationId, tableId
     addErrorMessage('An unexpected error has occurred, please refresh the page');
   }
 
-  useEffect(() => {
-    if (data && isSuccessPayload(data.createForkedStudio)) {
-      navigate(`/projects/${data.createForkedStudio.project.id}/edit`);
-      navigate(0);
-    }
-  }, [data]);
+  if (data && isSuccessPayload(data.createForkedStudio)) {
+    navigate(`/projects/${data.createForkedStudio.project.id}/edit`);
+    navigate(0);
+  }
```

Comment:
This should stay in a useEffect, it has a side effect so it cannot be performed during the rendering

Reply:
Done